CHANGES.rst

```diff
@@ -61,6 +61,8 @@ astropy.samp
 astropy.stats
 ^^^^^^^^^^^^^
 
+- Added ``circstd`` function to obtain a circular standard deviation.
```

Comment:
This might fix the change log failure.
```suggestion
- Added ``circstd`` function to obtain a circular standard deviation. [#10690]
```

Also need to add tests.